### PR TITLE
Revert "build(deps): bump vergen from 5.1.2 to 5.1.5"

### PR DESCRIPTION
**Pull request by teor2345**
Reverts ZcashFoundation/zebra#2074

#2074 conflicted with #2065

### CODE REVIEW

**Comment by teor2345** [comment]
I don't have admin-merge permissions any more, so @dconnolly can you approve this to unblock #2065?

**Review by dconnolly — APPROVED** [review]

**Pull request merged**
